lib/bricolage/s3datasource.rb

```diff
@@ -92,7 +92,7 @@ def load_s3cfg(path)
     end
     private :load_s3cfg
 
-    attr_reader :encryption
+    attr_reader :encryption, :region
```

Comment:
`:region` is already defined as an attr_reader here: https://github.com/bricolages/bricolage/blob/710a0cdef3d6c9e0c67faf8188ee4319ea4906de/lib/bricolage/s3datasource.rb#L38

Reply:
Thanks, fixed https://github.com/bricolages/bricolage/compare/710a0cdef3d6c9e0c67faf8188ee4319ea4906de..2668501e5b8210024460e860ae5f529af9b99b18